runtime/altair/src/migrations.rs

```diff
@@ -35,21 +40,30 @@ pub type UpgradeAltair1030 = (
 		crate::NativeToken,
 		crate::ExistentialDeposit,
 	>,
+	asset_registry::AssetRegistryMultilocationToXCMV3<crate::Runtime>,
+	// Low weight, mainly bumps storage version to latest (v1 to v2)
+	crate::DmpQueue,
+	// Low weight, mainly bumps storage version to latest (v2 to v3)
+	crate::XcmpQueue,
+	// Low weight, bumps uninitialized storage version from v0 to v1
+	pallet_xcm::migration::v1::MigrateToV1<crate::Runtime>,
 );
 
 /// The Upgrade set for Algol - it excludes the migrations already executed in
 /// the side releases that only landed on Algol (1028 & 1029) but not yet on
 /// Altair.
 #[cfg(feature = "testnet-runtime")]
-pub type UpgradeAltair1030 = (
-	runtime_common::migrations::nuke::Migration<crate::Loans, RocksDbWeight, 1>,
-	runtime_common::migrations::nuke::Migration<crate::InterestAccrual, RocksDbWeight, 0>,
-	pallet_rewards::migrations::new_instance::FundExistentialDeposit<
-		crate::Runtime,
-		pallet_rewards::Instance2,
-		crate::NativeToken,
-		crate::ExistentialDeposit,
-	>,
+pub type UpgradeAltair1031 = (
```

Comment:
Needed to bump bc Algol was already in 1031

Reply:
NOTE: Algol is currently on 1030.